### ISSUE TITLE
Preserving variables on profiles.

### ISSUE THREAD

**Issue by MrAtomik** [ALiVEOS/ALiVE.OS#569]
### Prerequisites

* [X] Can you reproduce the problem using ALiVE and CBA Only?
* [X] Are you running the latest version of ALiVE ?
* [X] Are you running the latest version of CBA_A3?
* [ ] Have you attached an unbinarized SQM?
* [ ] Have you attached your RPT (If the issue relates to data, also attch Server RPT and ALiVE Plugin Log)?
* [ ] Have you attached screenshots of the Parameters set on the offending Module?

### Description

Basically, ALiVE doesn't appear to be able to store variables on profiles when they are despawned. This would be a nice feature to add and would be very helpful.

### Steps to Reproduce

1. Set a variable on a vehicle with `setVariable`
2. Profile it.

**Expected behavior:** The variable to be saved and there when it spawns again.

**Actual behavior:** The variable is gone.

**Comment by friznit** [comment]
As per forum, there's a technique you can use to do this.  See the ALiVE wiki Script Snippets section.

**Comment by DavisBrown723** [comment]
5bb188f9149f4b5132df1678578074a4e9e0055a
